lib/hooks/kibela/hook.rb

```diff
@@ -0,0 +1,24 @@
+module Idobata::Hook
+  class Kibela < Base
+    screen_name 'kibe.la'
+    icon_url hook_image_url('icon.png')
```

Comment:
🙏  Sorry. Fix it. https://github.com/idobata/idobata-hooks/pull/75/commits/f9b5a88abced3a2214f57a04cff9c5f449932f50